app/controllers/audience_controller.rb

```diff
@@ -16,8 +16,16 @@ def index
   def export
     authorize :audience
 
-    audience_csv = Exports::AudienceExportService.new(current_seller).perform
-    send_data audience_csv, type: "text/csv"
+    options = params.required(:options)
+                 .permit(:followers, :customers, :affiliates)
+                 .to_h
+                 .transform_keys(&:to_s)
+
+    options.transform_values! { |v| ActiveModel::Type::Boolean.new.cast(v) }
```

Comment:
```suggestion
    options = params.required(:options)
                 .permit(:followers, :customers, :affiliates)
                 .to_h
```

IMO this is sufficient because:
- `transform_keys` is redundant as `to_h` returns an hash with indifferent access
- good thinking on `transform_values!`, though unnecessary since we're the sole consumer of this API